Soccar [Game-Server]/src/nl/soccar/gameserver/model/command/CommandListener.java

```diff
@@ -0,0 +1,49 @@
+package nl.soccar.gameserver.model.command;
+
+import nl.soccar.gameserver.controller.socnet.message.ChatMessage;
+import nl.soccar.gameserver.model.PlayerWrapper;
+import nl.soccar.gameserver.model.session.RoomWrapper;
+import nl.soccar.gameserver.model.session.SessionWrapper;
+import nl.soccar.library.enumeration.Privilege;
+
+/**
+ * @author Lesley
+ */
+public abstract class CommandListener {
+
+    private final Privilege minimumPrivilege;
+    private final boolean hostOnly;
+
+    public CommandListener(Privilege minimumPrivilege) {
+        this(minimumPrivilege, false);
+    }
+
+    public CommandListener(Privilege minimumPrivilege, boolean hostOnly) {
+        this.minimumPrivilege = minimumPrivilege;
+        this.hostOnly = hostOnly;
+    }
+
+    public abstract void execute(PlayerWrapper player, RoomWrapper room, Command command);
+
+    public final void executePrivileged(PlayerWrapper player, Command command) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)